js/foam/u2/Element.js

```diff
@@ -305,7 +306,49 @@ CLASS({
       name: 'innerHTML',
       transient: true,
       getter: function() { return this.outputInnerHTML(this.createOutputStream()).toString(); }
-    }
+    },
+    {
+      name: 'minWidth',
```

Comment:
If it were in a trait that only some views supported, then you have to know about that trait and make sure you have it set on anything you want to stack. That's possible, of course, but I think it's a needless complication. `Element`, like U1 `View`, contains a lot of things that are only needed sometimes.

On the other hand, Mark and I were talking about different styles of stacking and transitions anyway, so it may be that the current stacking style will change anyway, and these become unnecessary.